tools/windows/DatadogAgentInstaller/CustomActions/InstallOciPackages.cs

```diff
@@ -35,8 +35,21 @@ public InstallOciPackages(ISession session)
 
         private bool ShouldPurge()
         {
+            // Check KEEP_INSTALLED_PACKAGES first (takes precedence)
```

Comment:
Do we have to support PURGE and KEEP_INSTALLED_PACKAGES? Can we just replace PURGE?